dotCMS/src/main/java/com/dotmarketing/business/UserAPI.java

```diff
@@ -91,6 +91,8 @@
 	 */
 	public List<User> getUsersByName(String filter, int start,int limit, User user, boolean respectFrontEndRoles) throws  DotDataException;
 
+	public long getCountUsersByName(String filter) throws DotDataException;

```

Comment:
@john-thomas-dotcms @jgambarios @jcastro-dotcms, what should we do here? naming the new method to be grammatically correct _or_ follow the naming pattern of the other already existing methods?